test/Engine/FFITarget/TestHiddenMethods.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Autodesk.DesignScript.Runtime;
```

Comment:
There are already classes in FFITarget that have hidden base class and derived methods needed to be tested: https://github.com/DynamoDS/Dynamo/blob/f00b2f79c5accff55703469b4081b008066082d9/test/Engine/FFITarget/InheritenceTests.cs#L56
Let's try to avoid creating new dummy classes if possible.

Reply:
Nice catch, thanks